_typos.toml

```diff
@@ -0,0 +1,7 @@
+[files]
+# Don't correct submodule and json files
+extend-exclude = ["contracts/l1-l2-messaging/solidity/*", "*.json"]
+
+[default.extend-words]
+# Don't correct the word
+ede = "ede"
```

Comment:
Is this character sequence only encountered in hexadecimal strings? Is there really no smarter way around it than ignoring it?

One example of a false negative: someone wants to write "edge", but writes "ede" - this is not registered, right?

Reply:
I think I found a solution. What do you think about the following:
```toml
[files]
# Don't correct submodule and json files
extend-exclude = ["contracts/l1-l2-messaging/solidity", "crates/starknet-devnet-server/test_data"]
ignore-vcs = true

[default]
extend-ignore-re = ["0x[0-9a-fA-F]+"]
```

I added the regex, and `ignore-vcs`, and also modified the `extend-exclude` paths a bit.

Reply:
more options: https://github.com/crate-ci/typos/blob/master/docs/reference.md

Reply:
Awesome! I will try.